crates/oxc_linter/src/autofix/fixer.rs

```diff
@@ -356,170 +356,9 @@ mod test {
         ]);
         assert_eq!(result.fixed_code, TEST_CODE.replace("answer", "foo"));
         assert_eq!(result.messages.len(), 2);
+        // We should sort the error to pass the following assertion.
```

Comment:
What does this comment mean?

Reply:
Can you ignore this if it's not passing write now.

Reply:
Oh, you should add a `TODO:` here.

Reply:
It is already ignored. It means that we need to sort the remaining unfixed messages.

Reply:
Yes if it's easy to do so, let's do another PR for this after this.